src/commands/templates/install.js

```diff
@@ -77,11 +77,11 @@ class InstallCommand extends BaseCommand {
   }
 
   /**
-   * Setup Developer Console App Builder project from template install.yml configuration file
-   *
-   * @private
-   * @param {string} templateName
-   */
+     * Setup Developer Console App Builder project from template install.yml configuration file
+     *
+     * @private
+     * @param {string} templateName
+     */
```

Comment:
nitpick, the _previous_ annotations formatting was correct

Reply:
Fixed ✅ This was a mixup with the linter, all good now.